src/_includes/assets/css/screen.css

```
@@ -170,6 +182,135 @@ body {
 /* 5. Site Navigation
 /* ---------------------------------------------------------- */
 
+.exposed-button-nav {
+  color: var(--gray00);
+  background-color: var(--theme-color);
+  height: var(--header-element-size);
+  padding: 0 4px;
+  align-items: center;
+  min-width: var(--header-element-size);
+  border: 1px solid var(--gray00);
+  font-size: 18px;
+}
+
+@media (min-width: 601px) {
+  .exposed-button-nav {
+    padding-inline: 12px;
+  }
+}
+
+.exposed-button-nav:is(
+  :hover,
+  :hover:focus,
+  [aria-expanded='true'],
+  [aria-expanded='true']:hover
+),
+.lang-button-nav:is(
+  :hover,
+  :hover:focus,
+  [aria-expanded='true'],
+  [aria-expanded='false'].force-show
+) {
+  background-color: var(--gray00);
+  color: var(--theme-color);
+}
+
+.display-menu {
+  display: block !important;
+  max-height: calc(
+    100vh - var(--header-height) - var(--header-sub-element-size)
+  );
+  overflow-y: auto;
+  scrollbar-width: none;
+  text-align: left;
+  top: calc(var(--header-height) + var(--header-sub-element-size));
+}
+
+@media (min-width: 980px) {
+  .display-menu {
+    max-height: calc(100vh - var(--header-height));
+    top: var(--header-height);
+  }
+}
+
+ol,
+ul {
+  -webkit-padding-start: 40px;
+  margin-bottom: 10px;
+  margin-top: 0;
+  padding-inline-start: 40px;
+}
+
+.universal-nav {
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  -webkit-box-pack: justify;
+  -ms-flex-pack: justify;
+  align-items: center;
+  background-color: var(--theme-color);
+  color: var(--gray00);
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  font-size: 18px;
+  height: var(--header-height);
+  justify-content: space-between;
+  padding: 0 5px;
+}
+
+.nav-link {
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+  background-color: var(--gray90);
+  border: none;
+  color: var(--gray00);
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  height: 100%;
+  min-height: var(--header-sub-element-size);
+  padding: 2px 15px 0;
+  white-space: normal;
+  width: 100%;
+}
+
+.nav-link:focus-visible,
+.nav-link:not([aria-disabled='true']):hover {
+  background-color: var(--gray10);
+  color: var(--gray90);
+  cursor: pointer;
+  outline-color: transparent !important;
+  text-decoration: none;
+}
+
+.nav-link-flex {
+  -webkit-box-pack: justify;
+  -ms-flex-pack: justify;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  justify-content: space-between;
+}
+
+@media (min-width: 980px) {
+  .nav-list {
+    max-width: 15rem;
```

A lot of our base stylings come from an old Ghost theme, so `rem` doesn't work quite the same as it does in the main repo.

This can be set to `15em` or `270px` to match the main repo's dropdown width:

```suggestion
    max-width: 15em;
```